app/views/newPaymentFlow.scala.html

```diff
@@ -1,5 +1,17 @@
 <header role="banner">
   <a class="glogo" href="#"><img src="#" alt="The Guardian"></a>
+  <div class="countries">
```

Comment:
It's not too important here, but just as a heads up, it's probably a good idea to refer to these as ["country groups"](https://github.com/guardian/support-frontend/blob/master/assets/helpers/internationalisation/countryGroup.js) rather than countries. It fits in with conventions across the RR projects. @svillafe can probably provide more information.

Reply:
That's good to know! Will use that vocab when I move in React-land.